src/main/kotlin/com/openlattice/organizations/DatasetController.kt

```diff
@@ -176,8 +176,8 @@ class DatasetController : DatasetApi, AuthorizingComponent {
             @PathVariable(TABLE_NAME) tableName: String,
             @PathVariable(COLUMN_NAME) columnName: String,
             @RequestBody metadataUpdate: MetadataUpdate) {
-        ensureAdminAccess()
         val tableFqnToId = getExternalDatabaseObjectFqnToIdPair(organizationId, tableName)
+        ensureOwnerAccess(AclKey(tableFqnToId.second))
```

Comment:
Actually question for those who know more about these permissions. Should this be:
`AclKey(tableFqnToId.second, columnFqnToId.second)` instead?

Reply:
This is checking ownership of the table. If you were to add the column in it would be the second.

Reply:
Actually yeah, what drew said seems correct. Based on our slack discussion this should be ensureWriteAccess(...) on the two part AclKey

Reply:
FWIW updating entity set property metadata requires OWNER not WRITE (as does updating metadata of securable objects in general)

Reply:
ha, interesting.  What does WRITE on a table and no properties grant you as access then?

Reply:
Also, wrt Drew's comment: 
granting write on a column allows you to change the data, not the metadata.  There should be a separate level.  Our summer intern works on metadata and should not be allowed to change the data.
(if there would be work on a separate level, can we please merge this because it's blocking)

Reply:
> Also, wrt Drew's comment:
> granting write on a AclKey(table,column) allows you to change the data, not the metadata. There should be a separate level. Our summer intern works on metadata and should not be allowed to change the data.
> (if there would be work on a separate level, can we please merge this because it's blocking)

Intuition says that the level you're describing is `OWNER` 🤔